src/library/Uno.Material/Styles/Controls/v2/ToggleSwitch.xaml

```diff
@@ -0,0 +1,687 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:android="http://uno.ui/android"
+                    xmlns:ios="http://uno.ui/ios"
+                    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+                    xmlns:not_mobile="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:todo="todo"
+                    xmlns:toolkit="using:Uno.UI.Toolkit"
+                    mc:Ignorable="ios android todo">
+
+	<ResourceDictionary.ThemeDictionaries>
```

Comment:
theses resources are not part of the palette ?

Reply:
These were part of the color palette in v1 but its very specific and used only for ToggleSwitch. I don't want to clutter the SharedColors and SharedColorPalette with control-level resources. Should I keep it in the palette like in v1?

Reply:
For now keep it that way but keep a note to double-check with @NVLudwig tomorrow ;)